src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -75,6 +75,274 @@ curie_map:
   GC_ID: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/GC_ID/"
   SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
 
+extended_prefix_map:
+  - prefix: MONDO
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/MONDO_
+    uri_prefix_synonyms: []
+  - prefix: UMLS
+    prefix_synonyms: []
+    uri_prefix: http://linkedlifedata.com/resource/umls/id/
+    uri_prefix_synonyms: []
+  - prefix: NCIT
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/NCIT_
+    uri_prefix_synonyms: []
+  - prefix: DOID
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/DOID_
+    uri_prefix_synonyms: []
+  - prefix: EFO
+    prefix_synonyms: []
+    uri_prefix: http://www.ebi.ac.uk/efo/EFO_
+    uri_prefix_synonyms: []
+  - prefix: HGNC
+    prefix_synonyms: []
+    uri_prefix: http://identifiers.org/hgnc/
+    uri_prefix_synonyms:
+      - 'https://identifiers.org/hgnc:'
+      - 'https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:'
+  - prefix: HGNC_symbol
+    prefix_synonyms: []
+    uri_prefix: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
+    uri_prefix_synonyms: []
+  - prefix: HP
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/HP_
+    uri_prefix_synonyms: []
+  - prefix: SCTID
+    prefix_synonyms: []
+    uri_prefix: http://identifiers.org/snomedct/
+    uri_prefix_synonyms:
+      = http://snomed.info/id/
+  - prefix: OMIM
+    prefix_synonyms: []
+    uri_prefix: https://omim.org/entry/
+    uri_prefix_synonyms:
+      - http://identifiers.org/omim/
+      - http://purl.obolibrary.org/obo/OMIM_
+      - http://omim.org/entry/
+  - prefix: MESH
+    prefix_synonyms: []
+    uri_prefix: http://identifiers.org/mesh/
+    uri_prefix_synonyms: []
+  - prefix: Orphanet
+    prefix_synonyms: []
+    uri_prefix: http://www.orpha.net/ORDO/Orphanet_
+    uri_prefix_synonyms:
+      - https://www.orpha.net/ORDO/Orphanet_
+      - http://purl.obolibrary.org/obo/Orphanet_
+  - prefix: oboInOwl
```

Comment:
## Prefix synonyms
Handled these situations as well. `oio` is the only prefix synonym, which is on `oboInOwl`.